package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "orbital-sync",
   "description": "Synchronize multiple Pi-hole instances",
-  "version": "1.8.4",
+  "version": "1.9.0",
```

Comment:
This change will need to be removed and your commit message using conventional commits will automatically bump the version number